process/project_proposals.adoc

```diff
@@ -34,7 +34,8 @@ image::sandbox-process.png[Sandbox process]
      * After the meeting wraps up, the assigned TOC member may summarize the discussion on the PR in the form of a comment to document information for the project and community.
 . *Governance / legal issues*
    * CNCF staff handle governance / legal issues
-   * This can happen after TOC Sandbox Sponsors step forward - Lobbying for a sponsor is not permitted.
+   * This can happen after TOC Sandbox Sponsors step forward - Directly contacting a TOC member to serve as a project sponsor is not permitted.
```

Comment:
"Directly contacting a TOC member to serve as a project sponsor is not permitted." may be just adjust a bit to say "Please avoid contacting TOC members individually to serve as project sponsor"

Reply:
One note: we no longer require sponsors for Sandbox, these are for incubating + graduated only. This might confuse people if we leave it.